### ISSUE TITLE
problem of icon with spyder 2.3.0 beta4

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1786]
_From [pi...@synchrotron-soleil.fr](https://code.google.com/u/111976521841338120241/) on 2014-04-09T15:35:26Z_

When building the 2.3.0 beta4 debian package

I got this error

creating /tmp/buildd/spyder-2.3.0~beta4+dfsg/debian/python3-spyderlib/usr/share/pixmaps
error: can't copy 'img_src/spyder3.png': doesn't exist or not a regular file

And indeed img_src/spyder3.png does not exist in the source distribution

Is it possible to do a copy of spyder.png -> spyder3.png

then it will be possible to personalise the spyder3 icone.

thanks

Frederic

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1786_

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2014-04-09T19:31:03Z_

Thanks for the report. I forgot to add spyder3.png to our MANIFEST.in, sorry for that.

I just uploaded a new zipball to BitBucket with the same name but with this error fixed.

Lesson learned: I have to test our zipballs both in PY2 and PY3 (I just did in PY2 :)

**Labels:** MS-v2.3 Cat-Miscelleneous OpSys-Linux
